pyvista/core/dataset.py

```diff
@@ -202,6 +202,18 @@ def points(self) -> pyvista_ndarray:
     @points.setter
     def points(self, points: np.ndarray):
         """Set points without copying."""
+        if isinstance(points, pyvista_ndarray):
+            # simply set the underlying data
+            if points.VTKObject is not None:
+                try:
+                    self.GetPoints().SetData(points.VTKObject)
+                    self.GetPoints().Modified()
+                    self.Modified()
+                    return
+                except TypeError:
+                    pass
```

Comment:
Is this `TypeError` case for when `self.GetPoints() is None`? None of my local tests trip it (so it's uncovered).

Reply:
There are cases where VTKObject is None.  I've never encountered it, but I wanted a fallback option.  Should this fail, we use the legacy behavior.  Since we're not able to cover it, I say `# pragma: no cover` works.

Reply:
Hold on... when `VTKObject is None` we're already guarding with the `if`. Is there something else here then?

Reply:
Good point.  We are already guarding for that.  I'll remove this.